starter-code/server.js

```diff
@@ -23,8 +25,9 @@ app.get('/new', function(request, response) {
 
 app.get('/articles', function(request, response) {
   // REVIEW: This query will join the data together from our tables and send it back to the client.
-  // TODO: Write a SQL query which joins all data from articles and authors tables on the author_id value of each
-  client.query(``)
+  //Done! TODO: Write a SQL query which joins all data from articles and authors tables on the author_id value of each
+  client.query(`SELECT * FROM articles
+    JOIN authors ON articles.article_id=articles.author_id;`)
```

Comment:
We want to match up author ids. SO we want to set it so ON is looking for author_id on both articles and author(HINT :)